lib/tasks/biran_tasks.rake

```diff
@@ -12,7 +12,12 @@ namespace :config do
   config.files_to_generate.each do |file_name, options|
     desc %(Generate the #{file_name}#{options.fetch(:extension, '')} config file)
     task file_name do
-      config.create name: file_name, **options
+      begin
```

Comment:
use short syntax

Reply:
you can do it in any block... I think... maybe not... can you try it?

Reply:
Seems to work, although my syntax checker complains about a missing end, the code seems to work just fine